ontotext-yasgui-web-component/src/models/before-update-query-result.ts

```diff
@@ -1,4 +1,12 @@
 export interface BeforeUpdateQueryResult {
+  status: BeforeUpdateQueryResultStatus;
   message?: string;
   messageLabelKey?: string
+  parameters?: Record<string, string>[]
+}
+
+export enum BeforeUpdateQueryResultStatus {
```

Comment:
I'm OK with this name to some extent, but I wonder if it shouldn't be a bit more generic, like `QueryResponseStatus`?!

Reply:
Done